UnityProject/Assets/Scripts/UI/Systems/IngameMenu/LobbyUIPlayerListController.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using System;
+using Mirror;
 using UnityEngine;
 using TMPro;
 using ServerInfo;
 
-public class LobbyUIPlayerListController : MonoBehaviour
+public class LobbyUIPlayerListController : NetworkBehaviour
```

Comment:
We dont want to make UI componenets Network Behaviours, you'll need to make a custom network message.

Reply:
like this? https://github.com/unitystation/unitystation/pull/7764/commits/b2bd322e501cf223034a9fde1eaa0f2c81d84197